core/main/src/bootstrap/manifest/extn.rs

```diff
@@ -30,10 +30,12 @@ impl LoadExtnManifestStep {
 type ExtnManifestLoader = Vec<fn() -> Result<(String, ExtnManifest), RippleError>>;
 
 fn try_manifest_files() -> Result<ExtnManifest, RippleError> {
-    let dm_arr: ExtnManifestLoader = if cfg!(test) {
+    let dm_arr: ExtnManifestLoader = if cfg!(any(feature = "local_dev", feature = "pre_prod")) {
```

Comment:
What will be the expected differences between `local_dev` and `pre_prod`?

Reply:
Probably one for the future where we can configure local dev shell scripts and pre-prod environments differently.
`pre_prod` can be used by ci cd pipelines to generate builds.

We can configure other items for local_dev alone to make it customizable felt its a good tangent